internal/resource/config/oauth/client/oauth_client_resource.go

```diff
@@ -256,13 +257,17 @@ func (r *oauthClientResource) Schema(ctx context.Context, req resource.SchemaReq
 				Description: "Use this setting, for example, when you want to deploy a trusted application and authenticate end users via an IdP adapter or IdP connection.",
 				Computed:    true,
 				Optional:    true,
-				Default:     booldefault.StaticBool(false),
+				PlanModifiers: []planmodifier.Bool{
+					boolplanmodifier.UseStateForUnknown(),
+				},
 			},
 			"restrict_scopes": schema.BoolAttribute{
 				Description: "Restricts this client's access to specific scopes.",
 				Computed:    true,
 				Optional:    true,
-				Default:     booldefault.StaticBool(false),
+				PlanModifiers: []planmodifier.Bool{
+					boolplanmodifier.UseStateForUnknown(),
+				},
```

Comment:
What was the issue with these?

Reply:
These properties seemed to be always `false` in prior testing. When applying a different combination of attributes to make the correct ref in the authenticationApi/applications resource, this was not the case.